src/common/colors.ts

```diff
@@ -75,6 +75,7 @@ export const COLOR_MAP = {
     // rename these
     BASE: '#3567FD',
     DARK: '#002CB4',
+    DARKEST: '#000824',
```

Comment:
I chose this color randomly (I pulled up a color palette and used a color darker than `DARK`) so we might want to have design green-light this

Reply:
Yeah, let's get Josh to take a look before merging this.